addons/binding/org.openhab.binding.tesla/src/main/java/org/openhab/binding/tesla/handler/TeslaHandler.java

```diff
@@ -99,6 +99,7 @@
     public static final int CONNECT_RETRY_INTERVAL = 15000;
     public static final int MAXIMUM_ERRORS_IN_INTERVAL = 2;
     public static final int ERROR_INTERVAL_SECONDS = 15;
+    protected static final SimpleDateFormat DATE_FORMATTER = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.SSS");
```

Comment:
http://stackoverflow.com/questions/6840803/why-is-javas-simpledateformat-not-thread-safe

- So keep it local (not re-using)
- Use something else
- Make it Thread-local